ClientApp/viking-vault/src/UserDashboard/AutomaticPaymentList.tsx

```diff
@@ -102,16 +103,30 @@ class AutomaticPaymentList extends React.Component<any, IAutomaticPaymentsState>
         this.setState({payments : paymentList});
     }
 
+    toggleAutomaticPaymentState = (id: number, state:boolean) =>{
```

Comment:
Since you marked the function in the interface as being `void`, make it `void` here as well